spec/features/addition_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe 'Addition' do
+  it do
+    visit new_addition_path
+    fill_in 'Left', with: 1
+    fill_in 'Right', with: 2
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.